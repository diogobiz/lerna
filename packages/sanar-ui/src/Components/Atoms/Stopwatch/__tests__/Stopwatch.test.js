
import React from 'react'
import TestRenderer from 'react-test-renderer'

import ESStopwatch from '../'

it('renders correctly', () => {
    const component = (
        <ESStopwatch>
            <h1>ESStopwatch</h1>
        </ESStopwatch>
    )

    const tree = TestRenderer.create(component).toJSON()
    expect(tree).toMatchSnapshot()
})
