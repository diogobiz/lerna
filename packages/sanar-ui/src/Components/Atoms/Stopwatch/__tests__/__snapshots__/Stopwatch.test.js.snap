// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="es-stopwatch es-stopwatch--stopped"
>
  <i
    className="anticon es-eva-icon es-eva-icon--medium"
  >
    <i
      data-eva="pause-circle-outline"
      data-eva-fill="#fff"
      data-eva-height="18px"
      data-eva-width="18px"
    />
  </i>
  <span>
    00:00:00
  </span>
</div>
`;
